modules/top-level/output.nix

```diff
@@ -85,6 +85,7 @@ with lib;
         {
           inherit (config)
             extraPython3Packages
+            extraLuaPackages
```

Comment:
Can you explain the change and why it is specific to top-level outputs (rather than /modules/outputs.nix)?

Reply:
I've added a test, but I can't run it on my PC. Too much RAM is required. Is there a way to run only a single test? I want to check if it fails before this change and passes after.

Reply:
> Is there a way to run only a single test?

Not that I'm aware of. That'd be nice though, my laptop takes around an hour to run the basic tests.

Letting buildbot do the work is fine if your hardware can't handle it.

Reply:
> > Is there a way to run only a single test?
> 
> Not that I'm aware of. That'd be nice though, my laptop takes around an hour to run the basic tests.

I found it.

```
nix build .#checks.x86_64-linux.tests.passthru.entries.modules-output-extraLuaPackages
```

will run only this test. So I checked that it fails before the fix (`require` returns `module 'jsregexp' not found` error).

Reply:
Thanks for the tip using `tests.passthru`, that'll come in handy!



> Can you explain the change and why it is specific to top-level outputs (rather than /modules/outputs.nix)?

I've edited the description to clarify the goal of the PR, please comment/edit further if my changes are inaccurate.